crypto/fipsmodule/curve25519/curve25519.c

```diff
@@ -53,25 +73,28 @@
 // For Ed25519, dom2(F,C) is the empty string and PH the identify function,
 // cf. rfc8032 5.1.
 
-void ed25519_sha512(uint8_t out[SHA512_DIGEST_LENGTH],
-  const void *input1, size_t len1, const void *input2, size_t len2,
-  const void *input3, size_t len3) {
-
+void ed25519_sha512(uint8_t out[SHA512_DIGEST_LENGTH], const void *input1,
+                    size_t len1, const void *input2, size_t len2,
+                    const void *input3, size_t len3, const void *input4,
+                    size_t len4) {
   SHA512_CTX hash_ctx;
   SHA512_Init(&hash_ctx);
   SHA512_Update(&hash_ctx, input1, len1);
   SHA512_Update(&hash_ctx, input2, len2);
   if (len3 != 0) {
     SHA512_Update(&hash_ctx, input3, len3);
   }
+  if (len4 != 0) {
+    SHA512_Update(&hash_ctx, input4, len4);
+  }
   SHA512_Final(out, &hash_ctx);
 }
 
 // Public interface functions
 
 void ED25519_keypair_from_seed(uint8_t out_public_key[ED25519_PUBLIC_KEY_LEN],
-  uint8_t out_private_key[ED25519_PRIVATE_KEY_LEN],
-  const uint8_t seed[ED25519_SEED_LEN]) {
+                               uint8_t out_private_key[ED25519_PRIVATE_KEY_LEN],
```

Comment:
Running clang-format once on everything and then enforcing it in our CI is on my wish list for 2025. Until then it would be nice to keep formatting changes minimal or in separate PRs to make it easier to review actual code changes.